gluefactory/models/deeplsd_inference.py

```diff
@@ -171,7 +171,7 @@ def detect_afm_lines(
 
         # Merge close-by lines together
         if merge:
-            lines = merge_lines(lines, thresh=4, overlap_thresh=0).astype(np.float32)
+            lines = merge_lines_torch(torch.from_numpy(lines).to(img.device), thresh=4, overlap_thresh=0).cpu().numpy()
```

Comment:
I rememeber I got an error with this line yesterday. Is this resolved by your changes? @FelixMoeller3

Reply:
Did you get an error using `merge_lines` or using `merge_lines_torch`?

Reply:
@hardik01shah what is this change about? Should we rather use merge_lines for now?

Reply:
@hardik01shah push

Reply:
We should use merge_lines. That is the original deeplsd implementation. This got messed up in the merge somehow

Reply:
Fixed now.